### PR TITLE
Fix README.md Markdown

### LINKED ISSUE #137: Readme broken
I'm surprised that nobody seems to have noticed this in the past ~14 years, but the Readme file is totally broken. Formattings are all over the place and thousands of lines of text are simply not visible because they go over the 512K character limit (which I just today learned that it exists on Github).
I looked at the history and found changes from 2006 that already had this long Readme file. Either something about Github was different back then or it was already broken back then.

Example screenshot:
image

How the end looks:
image

The file actually goes on beyond that, but it's not displayed.

### PR DESCRIPTION
**Pull request by genpfault**
Fixes #137

### CODE REVIEW

**Comment by bk138** [comment]
Thanks for taking on this! Could you also please add in the commit message what you changed/added/removed?

File: README [hunk 1]
```diff
@@ -8,739 +8,742 @@ This repo represents the x11vnc codebase that once resided in the LibVNCServer r
 It was split off LibVNCServer and moved here with
 https://github.com/LibVNC/libvncserver/commit/498d222976975f53dea885cfe43ef0f805abd412.
 
-Copyright (C) 2002-2010 Karl J. Runge <runge@karlrunge.com>
```
Comment (bk138) [inline README]: the copyright notice should be left intact

File: README [hunk 2]
```diff
@@ -8,739 +8,742 @@ This repo represents the x11vnc codebase that once resided in the LibVNCServer r
...
+These gateway access modes also can be done automatically for you via
+the "Proxy/Gateway" setting in SSVNC (including the Chaining SSH's
+case, "Double Proxy".)
+
+## Chaining SSH's
+
+Note that for use of a ssh gateway and -L redirection
+to an internal host (e.g. "-L 5900:otherhost:5900") the VNC traffic
+inside the firewall is not encrypted and you have to manually log into
+otherhost to start x11vnc. Kyle Amon shows a method where you chain
```
Comment (bk138) [inline README]: this is an addtition?

File: README [hunk 3]
```diff
@@ -780,17557 +783,468 @@ export PATH CPPFLAGS LDFLAGS
...
+On Solaris using static archives libssl.a and libcrypto.a instead of
+.so shared libraries (e.g. from www.sunfreeware.com), we found we
+needed to also set LDFLAGS as follows to get the configure to work:
+
+```
 env LDFLAGS='-lsocket -ldl' ./configure --with-ssl=/path/to/openssl ...
 make
-  ```
-
-#   Beta Testing:
```
Comment (bk138) [inline README]: yeah, beta test and 0.9.13 can go away

File: README [hunk 4]
```diff
@@ -780,17557 +783,468 @@ export PATH CPPFLAGS LDFLAGS
...
-     * An environment variable, X11VNC_SB_FACTOR, allows one to scale the
-       -sb screenblank sleep time from the default 2 secs.
-     * An experimental option -unixsock is available for testing. Note,
-       however, that it requires a manual change to
-       libvncserver/rfbserver.c for it to work.
-     * Documented that -grabkbd is no longer working with some/most
-       window managers (it can prevent resizing and menu posting.)
-
-
-   Here are some features that appeared in the 0.9.12 release (Sep/2010):
```
Comment (bk138) [inline README]: those should be synced with NEWS probably

**Review by bk138 — CHANGES_REQUESTED** [review]

File: README (hunk 2 above)
Comment (genpfault) [reply README]: Nope, this "Chaining SSH's" was moved here from line 9308 in the old README.

Original source was http://www.karlrunge.com/x11vnc/chainingssh.html

File: README (hunk 4 above)
Comment (genpfault) [reply README]: I'll add a top-level 'News' section between 'Annoucements' & 'FAQs' & drop a link to the NEWS file there.

**Pull request merged**